Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -52,22 +52,53 @@
 module Homebrew
   module_function
 
-  def audit
-    Homebrew::CLI::Parser.parse do
-      switch      "--strict"
-      switch      "--online"
-      switch      "--new-formula"
-      switch      "--fix"
-      switch      "--display-cop-names"
-      switch      "--display-filename"
-      switch      "-D", "--audit-debug", description: "Activates debugging and profiling"
+  def audit_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `audit` [<options>] [<formulae>]:
+
+        Check <formulae> for Homebrew coding style violations. This should be
+        run before submitting a new formula.
+
+        If no <formulae> are provided, all of them are checked.
+
+      EOS
+      switch      "--strict",
```

Comment:
Sorry, terrible pedantry now but given the way `description` is I'd be tempted to say that `switch "--strict",` would look better.